src/test/xsl/preprocess/conrefImpl.xspec

```diff
@@ -55,5 +55,114 @@
     </x:scenario>    
     
   </x:scenario>
+  
+  <!--
```

Comment:
This works in Oxygen but not in our unit test driver. Since this PR doesn't change this template, keeping as out commented code